lib/mocha.js

```diff
@@ -505,7 +505,9 @@ Mocha.prototype._growl = growl.notify;
  * mocha.globals(['jQuery', 'MyLib']);
  */
 Mocha.prototype.globals = function(globals) {
-  this.options.globals = (this.options.globals || []).concat(globals);
+  this.options.globals = this.options.globals || [];
+  globals = globals || [];
+  this.options.globals = this.options.globals.concat(globals);
   return this;
```

Comment:
Since we shouldn't allow empty strings either...
```js
Mocha.prototype.globals = function(globals) {
  this.options.globals = (this.options.globals || []).concat(globals).filter(Boolean);
  return this;
};
```